src/WebJobs.Script.WebHost/Security/KeyManagement/BlobStorageSecretsRepository.cs

```diff
@@ -65,15 +68,17 @@ public override bool IsEncryptionSupported
             }
         }
 
-        protected virtual CloudBlobContainer CreateBlobContainer(string connectionString)
+        protected virtual BlobContainerClient CreateBlobContainerClient(string connectionString)
```

Comment:
The parameter name is the connection string and it seemed to have been used as a connection string in a previous version of the code. Are we sure we are doing the right thing here?

Reply:
Yes. BlobStorageSecretsRepository is called in 2 places:
- BlobStorageSasSecretsRepository inherits from it, but overrides the CreateBlobContainerClient() method to create a container from a SasUri
- DefaultSecretManagerProvider creates a BlobStorageSecretsRespository object and now passes in a Connection name instead of a connection string.